src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -441,6 +445,9 @@
     /** Whether to apply the ellipsoid->geoid offset to all elevations in the response */
     public boolean geoidElevation = false;
 
+    /** Which path comparator to use */
+    public String pathComparator = null;
+
```

Comment:
I cannot any places where `pathComparator` is set. I think a comment on how this is used(set to somethingother `null`) should be documented. Is the plan to integrate with some kind of configuration later, routing request parameter, or just fork the code and inject a different strategy?

Reply:
It's not set explicitly in the code, but if you include configuration in `router-config.json`, it will get set automatically, e.g.

    // router-config.json
    {
      "routingDefaults": { 
         "pathComparator":"duration" 
       }
    }

This happens via the existing config-setting code:

https://github.com/opentripplanner/OpenTripPlanner/blob/adb45efafb6e44e7add459e6a12c166d4abd427f/src/main/java/org/opentripplanner/standalone/Router.java#L98

I'll add code to allow this to be set via an API parameter as well.

Reply:
Thank you for the clarification. I think your solution is the best, it might be confusing to have "config" params only.